modify-publication/src/main/java/no/unit/nva/publication/modify/ModifyPublicationHandler.java

```diff
@@ -20,6 +21,7 @@
 import java.util.UUID;
 
 import static nva.commons.utils.JsonUtils.objectMapper;
+import static org.apache.http.HttpHeaders.CONTENT_RANGE;
 
 public class ModifyPublicationHandler extends ApiGatewayHandler<UpdatePublicationRequest, PublicationResponse> {
```

Comment:
Could maybe ApiGatewayHandler itself do the verifying?

default method and ensure we call super?

like: if method.PUT && !validateNotPartialContentReuqest  ; bad boy.

Reply:
Maybe we should have have a PutRequestHander and a (GetRequestHandler etc.) where each one of them checks for valid requests in their context (PUT,GET,POST, etc) requests. So we can void the clutter of 
```
if(method.name==something){
 validate()
}